client/components/site-title/index.jsx

```diff
@@ -45,17 +45,19 @@ class SiteTitleControl extends React.Component {
 				<FormFieldset>
 					<FormLabel htmlFor="blogname">{ this.props.translate( 'Site Title' ) }</FormLabel>
 					<FormTextInput
+						autoFocus
 						name="blogname"
-						value={ this.props.blogname }
 						onChange={ this.onChangeSiteTitle }
+						required
+						value={ this.props.blogname }
 					/>
 				</FormFieldset>
 				<FormFieldset>
 					<FormLabel htmlFor="blogdescription">{ this.props.translate( 'Tagline' ) }</FormLabel>
```

Comment:
I'm equal to this, but just mentioning it for consideration: in our mockups this field is called "Site Description".

Reply:
Yeah; still thinking of sticking with the 'Tagline' wording, see https://github.com/Automattic/wp-calypso/pull/21439#discussion_r160961255